changelog/README.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 12th November 2025
+* Updated rate limits per environments in [Environments](../guidelines/environments.md#request-limits). Documentation-only, no change to API:
+  * 200 requests per `AccessToken` within 30 seconds for Production environment.
+  * 200 requests per `AccessToken` within 30 seconds for Demo environment.
+  * Removed rate limiting per endpoint per `AccessToken`. 
```

Comment:
suggestion: I'd mention "updated rate limits documentation" and removed the actual values from the changelog. Also, claiming "Documentation-only, no change to API" is sorta controversial.


```suggestion
* Updated rate limits documentation in [Environments](../guidelines/environments.md#request-limits).
* Removed rate limiting per endpoint per `AccessToken`.
```